packages/typing_extensions/meta.yaml

```
@@ -0,0 +1,9 @@
+package:
+  name: typing_extensions
```

The package name is 
```suggestion
  name: typing-extensions
```
https://pypi.org/project/typing-extensions/